### ISSUE TITLE
Добавление стороннего сайта в модуль

### ISSUE THREAD

**Issue by haydei** [Waujito/youtubeUnblock#78]
Всем привет. Подскажите можно ли добавить сторонние сайты помимо ютуба для разблокировки в установленный модуль OpenWRT? Если да то подскажите как.

**Comment by Waujito** [comment]
Да, можно дописать в --sni-domains. Можно написать --sni-domains=all чтобы обходить всё.

**Comment by haydei** [comment]
Все получилось спасибо
